transform/mattermost-analytics/models/marts/product/dim_daily_server_config.sql

```diff
@@ -1,496 +1,352 @@
 {{ config(
     tags=["disabled"]
```

Comment:
Let's remove this tag from this model in order for the model to run on this branch. This will help get an estimate of how long the first run will take.

Reply:
Also let's make sure that this is explicitly running on a large warehouse. XS might look cheaper, but spilling to disk is likely to be causing longer run times.